### DISCUSSION TITLE
Column Summary Doesn't work when using Collections.

### DISCUSSION THREAD

**Discussion by abhaypithadiya** [Power-Components/livewire-powergrid#1554] [BUG]
I have a datasource which returns a collection.

```
    public function datasource(): ?Collection
    {
        return $this->productCollection;
    }
```

When I use any column summary it doesn't display.

```
            Column::make('Gross Wt', 'gross_wt')
                ->withSum('Total Gross Wt', true, false)
                ->searchable()
                ->sortable(),
```

Does this feature only work for Eloquent? I couldn't find anything in the documentation.

**Comment by dansysanalyst** [comment]
Thanks for opening the discussion @abhaypithadiya. I guess it had slipped through our fingers.

@luanfreitasdev has submitted a PR https://github.com/Power-Components/livewire-powergrid/issues/1557

Many thanks Luan!
________
⭐ Please consider starring our repository. It is a great way to support the project.
📢 Share PowerGrid with your dev-community, friends and colleagues: Tweet, write a blog article or record a review video. 
👥 PowerGrid is free, open-source and maintained by the authors and all contributors.

**Reply by abhaypithadiya** [reply]
Happy to Help! Thank you @dansysanalyst @luanfreitasdev 🔥
